ci/terraform/waf.tf

```diff
@@ -1,3 +1,22 @@
+# The IP address blocks below are referenced from here:
+# https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/working-at-gds/gds-internal-it/gds-internal-it-network-public-ip-addresses
+resource "aws_wafv2_ip_set" "gds_ip_set" {
+  name               = "${var.environment}-gds-ip-set"
+  scope              = "REGIONAL"
+  ip_address_version = "IPV4"
+
+  addresses = [
+    "217.196.229.77/32",
+    "217.196.229.79/32",
+    "217.196.229.80/32", # (BYOD VPN Only)
+    "217.196.229.81/32",
+    "51.149.8.0/25",   # (GDS and CO VPN)
+    "51.149.8.128/29", # (BYOD VPN only)
+  ]
```

Comment:
I would Suggest to add old IPs as well here , staging stubs are still using  OLD IPs

Reply:
Thanks @pskushwaha1 - I've added them just in case